base/cvd/cuttlefish/host/commands/cvd/server_command/power.cpp

```diff
@@ -93,6 +104,19 @@ class CvdDevicePowerCommandHandler : public CvdServerHandler {
     return valid_ops;
   }
 
+  Result<std::string> SummaryHelp() const override { return kSummaryHelpText; }
+
+  bool ShouldInterceptHelp() const override { return false; }
+
+  Result<std::string> DetailedHelp(
+      std::vector<std::string>& arguments) const override {
+    std::string replacement = "<command>";
+    if (!arguments.empty()) {
+      replacement = arguments.front();
+    }
+    return fmt::format(kDetailedHelpText, replacement);
```

Comment:
Not a fan of the format string with the `{}` being 70 lines away from what is filled in. Can `kDetailedHelpText` be moved closer, or inlined into this call?

Reply:
Done